### PR TITLE
fixes: #575 links to change selected Repo

### PR DESCRIPTION
**Pull request by on2onyekachi**
fixes: #575 
Before: 

https://user-images.githubusercontent.com/66228222/168657818-eb2f8841-a174-4dc2-8ec6-e926787688ed.mov

After:

https://user-images.githubusercontent.com/66228222/168657923-d3087f53-a236-45f7-9f9f-8d744b70fd7a.mov

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

* [x] tests pass -- look for a green checkbox ✔️ a few minutes after opening your PR -- or run tests locally with `rake test`
* [x] code is in uniquely-named feature branch and has no merge conflicts
* [x] PR is descriptively titled
* [x] PR body includes `fixes #0000`-style reference to original issue #
* [x] ask `@publiclab/reviewers` for help, in a comment below

> We're happy to help you get this ready -- don't be afraid to ask for help, and **don't be discouraged** if your tests fail at first!

If tests do fail, click on the red `X` to learn why by reading the logs.

Please be sure you've reviewed our contribution guidelines at https://publiclab.org/contributing-to-public-lab-software

We have a loose schedule of reviewing and pulling in changes every Tuesday and Friday, and publishing changes on Fridays. 

Thanks!

### CODE REVIEW

**Comment by TildaDares** [comment]
@on2onyekachi When I tried it locally, it redirected to another URL entirely. We want only the repo to change.

https://user-images.githubusercontent.com/63427719/168799052-87aa7090-7a04-4dae-ab0b-b0b34a54ac39.mov

**Comment by on2onyekachi** [comment]
I see. I also did experience this while working locally. I guess that is a result of the hard-coded URL from the [config.json](https://github.com/publiclab/community-toolbox/blob/main/config.json) file. I had to change the URL pointing to my local server URL to 127.0.0.1 before actually working on it. 
Screenshot 2022-05-17 at 14 58 59

Maybe we could get a solution that should allow the use of absolute URLs/URIs that work locally in a local development environment and also work on the remote server without modification.?

**Comment by TildaDares** [comment]
@on2onyekachi Let's take the easy route and leave only the changes in `index.html` and `configure.js`. The rest can be removed. Thanks!!

**Comment by on2onyekachi** [comment]
> @on2onyekachi Let's take the easy route and leave only the changes in `index.html` and `configure.js`. The rest can be removed. Thanks!!

Okay, though that's like going back to the default behavior of both links. Maybe, is there a much better approach to this?. 

I also think due to the hash(#) symbol on the URL it can't possibly reload the page, hence the manual reloading of the page on the select repository dropdown buttons.    
Screenshot 2022-05-17 at 21 54 40

@TildaDares , Just before I remove the other files I will want to propose we take out that line in the [configure.js](https://github.com/publiclab/community-toolbox/blob/main/src/scripts/configure.js) file since we're only making use of it for this case only and work directly with the HTML href attribute. `href="/#-all"`. Thereby having an absolute URL that will work on a local environment and also on a server.  This should work to fix the redirecting URL and also change the selected repo to all.

**Comment by on2onyekachi** [comment]
here is how it is.

https://user-images.githubusercontent.com/66228222/168928209-dcb73197-6cd7-4341-8a42-f588e63ef616.mov

**Comment by on2onyekachi** [comment]
Yeah!!  Thanks, @TildaDares 🙂

File: examples/demo.js [hunk 1]
```diff
@@ -188,8 +188,10 @@ document.addEventListener('DOMContentLoaded', function () {
             })
         })
 
-
-
+        $('.org-wide-issues').on('click', ()=>{
+            location.href = $('.org-wide-issues').attr("href")
+            location.reload();
+        })
```
Comment (TildaDares) [inline examples/demo.js:194]: This works perfectly @on2onyekachi!! Can you just make sure that the indentation is two spaces deep? Thank you!!

Comment (on2onyekachi) [reply examples/demo.js:194]: Thanks, @TildaDares No, I followed the indentation of the file, all 4spaces. Should I fix all of it, or make it an FTO?

**Comment by on2onyekachi** [comment]
Can I just fix just that? @TildaDares  thanks.

File: examples/demo.js (hunk 1 above)
Comment (TildaDares) [reply examples/demo.js:194]: You know what let’s leave it that way so it doesn’t look odd.

Comment (on2onyekachi) [reply examples/demo.js:194]: OKAYY

**Comment by TildaDares** [comment]
Thank you for this @on2onyekachi!!!

**Pull request merged**